examples/platform/silabs/MatterConfig.cpp

```diff
@@ -41,6 +41,12 @@
 #include "MemMonitoring.h"
 #endif
 
+#ifdef SIWX_917
+extern "C" {
+int32_t wfx_wifi_rsi_init(void);
```

Comment:
this should be removed

Reply:
Hi @jepenven-silabs 

Let me know where should i keep this declaration.

I kept this declaration in MatterConfig.h earlier then you told me to eliminate from there,

If I keep in any wifi header files then it has lot of other header file dependency, I cannot create a new header just for this declaration.

Reply:
So where is the implementation of this function? This is a odd way of doing things. The prototype should be in a relevant header to where it is implemented. Not where it is used. You should include the header that owns that prototype/ implementation.

Reply:
ok added appropriate header file